lib/fluent/plugin/in_debug_agent.rb

```diff
@@ -31,6 +32,11 @@ def initialize
 
     def configure(conf)
       super
+      if @unix_path
+        unless ::Fluent::FileUtil.writable?(@unix_path)
+          raise ConfigError, "in_debug_agent: `#{@unix_path}` is not writable."
```

Comment:
`.` is not needed when error message is one sentence.
This is consistency for other places.